_episodes_rmd/05-r-markdown.Rmd

```diff
@@ -35,44 +31,109 @@ set.seed(20210109)
 ### Contents
 1. [R for data analysis review](#r-for-data-analysis-review)
 1. [What is R Markdown and why use it?](#why-use-r-markdown)
-1. [Creating a reports directory](#creating-a-reports-directory)
 1. [Creating an R Markdown file](#creating-an-r-markdown-file)
 1. [Basic components of R Markdown](#basic-components-of-r-markdown)
     + [Header](#header)
     + [Code chunks](#code-chunks)
     + [Text](#text)
 1. [Starting the report](#starting-the-report)
+    + [Change knit directory](#change-knit-directory)
+    + [Add code](#add-code)
+    + [Plot](#plot)
+    + [Table](#table)
+    + [Knitting](#knitting)
+1. [Customizing the report](#customizing-the-report)
+    + [Table format](#table-format)
+    + [Messages](#messages)
+    + [Code](#code)
+    + [Bonus](#bonus)
 1. [Formatting](#formatting)
+    + [Headers](#headers)
+    + [Lists](#lists)
+    + [Hyperlinks](#hyperlinks)
 1. [Applying it to your own data](#applying-it-to-your-own-data)
 
 ## R for data analysis review
 
-Remember that yesterday we made a scatter plot of year vs. population, separated into a plot for each contient, 
-and that it had 2 outliers? Which countries are those?
+Remember that yesterday we made a scatter plot of year vs. population, separated into a plot for each contient, and that it had 2 outliers:
+
+```{r, message = FALSE}
+library(tidyverse)
+smoking <- read_csv('data/smoking_cancer.csv')
+
+smoking %>% 
+  ggplot(aes(x=year,y=pop)) +
+  geom_point() +
+  facet_wrap(vars(continent))
+```
+
+Write some code to figure out which countries these are (even if you already know!).
+
+> ## Solution
+> ```{r}
+> smoking %>% filter(pop > 5e8) %>% select(country) %>% unique()
```

Comment:
I've been using distinct() in the data cleaning lesson, since it's a dplyr function. In this application, it doesn't matter which one you use...
```suggestion
> smoking %>% filter(pop > 5e8) %>% select(country) %>% distinct()
```